packages/components/avatar/src/avatar-icon.tsx

```diff
@@ -11,6 +11,7 @@ export const AvatarIcon: FC<IconProps> = (props) => (
     className="ui-avatar__icon"
     {...props}
   >
+    <title>Avatar Icon</title>
```

Comment:
Instead of adding a `title` here, add an `aria-label` using AvatarImage.

`avatar.tsx`
```diff
if (isFallback)
  return name ? (
    <AvatarName name={name} format={format} />
  ) : (
-     cloneElement(icon, { role: "img" })
+     cloneElement(icon, { role: "img", "aria-label": alt ?? "Avatar icon" })
  )
```